core/src/main/java/com/novoda/accessibility/ActionMenuItem.java

```diff
@@ -0,0 +1,273 @@
+package com.novoda.accessibility;
+
+import android.content.Intent;
+import android.content.res.Resources;
+import android.graphics.drawable.Drawable;
+import android.support.annotation.Nullable;
+import android.support.annotation.StringRes;
+import android.util.Log;
+import android.view.ActionProvider;
+import android.view.ContextMenu;
+import android.view.MenuItem;
+import android.view.SubMenu;
+import android.view.View;
+
+class ActionMenuItem implements MenuItem {
```

Comment:
In both this and `ActionsMenu`, I used `UnsupportedOperationException` for the methods that where:

- we don't use/need them
- they're not called by `MenuInflater` during inflation

Reply:
*were 🙀